Makefile

```diff
@@ -397,34 +397,35 @@ ci-clean:
 	podman $(PODMAN_REMOTE_ARGS) \
 	    image prune --all --filter="label=aro-*=true"
 
+# Define OUTPUT_DIR at the top, so it works locally and in Azure pipelines
+OUTPUT_DIR ?= $(shell pwd)
+
+# Define OUTPUT_DIR at the top, so it works locally and in Azure pipelines
+OUTPUT_DIR ?= $(shell pwd)
```

Comment:
q: how is this used and why is it important?

Reply:
This isn't necessary anymore; I had added it for testing purposes, but we can remove it now. Apologies for any confusion